src/__tests__/index-test.tsx

```diff
@@ -0,0 +1,39 @@
+import { expect } from "chai";
+
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import * as Enzyme from "enzyme";
+import * as Sinon from "sinon";
+
+const RegistryAdmin = require("../index");
+import { BrowserRouter } from "react-router-dom";
+import App from "../components/App";
+
+describe("RegistryAdmin", () => {
+  let renderSpy;
+
+  beforeEach(() => {
+    renderSpy = Sinon.spy(ReactDOM, "render");
+    new RegistryAdmin({ username: "", imgSrc: "" });
```

Comment:
The `imgSrc` is used for testing. The default value is the path to the logo in the repo but making it an empty string for tests makes mocha happy (it cannot call `require(''...")`).